tests/features/hashing/test_hashers.py

```diff
@@ -31,3 +31,12 @@ def test_argon2_needs_rehash(self):
         # Here argon2 method is invoked without custom options and will
         # use default argon2 configuration, so rehash will be needed
         assert Hash.needs_rehash(hashed, driver="argon2")
+
+    def test_should_return_hash_content_as_string(self):
+
+        drivers = ["argon2", "bcrypt"]
```

Comment:
To avoid hard-coding drivers list, you could use this:
```suggestion
        drivers = Hash.drivers.keys()
```
(It's equivalent to `self.application.make("hash").drivers.keys()`

Reply:
Great!!